.config/ags/modules/.configuration/user_options.js

```diff
@@ -56,6 +56,9 @@ let configOptions = {
             'default': "auto",
         },
     },
+    'configPaths': {
```

Comment:
idk why but i don't like this section name

Reply:
what would you name it? I tried to make it a group where other custom paths fit as well if ever needed